ee/debug/checkups/logs.go

```diff
@@ -46,6 +47,12 @@ func (c *Logs) Run(_ context.Context, fullFH io.Writer) error {
 	defer logZip.Close()
 
 	matches, _ := filepath.Glob(filepath.Join(c.k.RootDirectory(), "debug*"))
+
+	if runtime.GOOS == "darwin" {
+		stdMatches, _ := filepath.Glob("/var/log/kolide-k2/*")
```

Comment:
I don't love this in `logs`. I kinda want it in another directory. But I can live with it.

Reply:
Well, I'll go ahead and merge this one and I can move it to another directory when I add stderr/out for other platforms. Should we just call the dir stderr?

Reply:
If we make a separate flare for it, I'd call it `InitLogs` or something. Because it collects the logs from the init system

Reply:
But 🤷 maybe the windows side isn't quite that simple.